samples/nrf9160/aws_fota/README.rst

```diff
@@ -31,56 +112,133 @@ Building and running
-
-
-7. Choose **Snapshot** as job type and press **Next**.
+.. _configuring:
+
+Sample configuration
+====================
+
+Before you build the sample, check and update the following configuration options:
+
+Application version:
+   The version string is printed when the sample starts.
+   (?not implemented yet?)
+   Use this information to verify that the FOTA update worked.
+
+.. _security_tag:
+
+Secure tag for TLS credentials:
+   (?should be security tag?)
+
+   By default, the sample uses the preconfigured (?preprogrammed? pre-stored?) certificates for nRF Cloud.
+   To use different certificates, configure a different security tag.
+   If you used LTE Link Monitor to store the certificates, make sure to configure the security tag that you used to store them.
+
+AWT IoT MQTT broker name:
+   By default, the sample uses nRF Cloud's MQTT broker.
+   Change this value to AWS IoT's MQTT broker.
+   To find the address of the AWS IoT MQTT broker, open the AWS IoT console, go to **Test** and select **View endpoint** from the **Connected as XXX** drop-down menu.
+
+Custom MQTT Client Id:
+   The client ID links your board to the thing in AWS IoT.
+   By default, the client ID is ``nrf-IMEI``, where *IMEI* is the IMEI number of your board.
+   If you chose a different name for your thing in AWS IoT, check this option and specify the AWS IoT thing name as client ID.
+
+Use provisioned certificates:
+   If this option is checked, the sample uses the certificates that are stored in the modem.
+   Uncheck this option if you want to use the certificates added to the ``certificates.h`` file.
+   The sample then stores these certificates with the security tag that is defined as :ref:`Secure tag for TLS credentials <security_tag>`.
+
+   .. warning::
+      The sample will overwrite existing certificates that are stored with the configured security tag.
+
+For all other values, use the default values unless you are using a custom MQTT server.
+
+Testing
+=======
+
+After programming the sample to the board, test it by performing the following steps:
+
+1. |connect_terminal|
+#. Reset the board.
+#. Confirm that the sample prints the configured application version and connects to AWS IoT.
+   You should see output like the following::
+
+      ***** Booting Zephyr OS build v1.14.99-ncs3-snapshot2-1281-g40b430ba977c *****
+
+      The MQTT AWS Jobs FOTA Sample
+      nrf_inbuilt_key_delete(42, 0) => result=0
+      nrf_inbuilt_key_delete(42, 1) => result=0
+      nrf_inbuilt_key_delete(42, 2) => result=0
+      nrf_inbuilt_key_write => result=0
+      nrf_inbuilt_key_write => result=0
+      nrf_inbuilt_key_write => result=0
+      LTE Link Connecting ...
+      LTE Link Connected!
+      IPv4 Address 0xeeb73a0d
+      client_id: nrf-aws-fota
+      [mqtt_evt_handler:129] MQTT client connected!
+
+   (?any data in there that should not be public or that is confusing?)
```

Comment:
For me the IPv4 address is a bit confusing since it's a IPv4 address converted to hex in big-endian(I think) Usually IPv4 addresses are formatted in <0-255>.<0-255>.<0-255>.<0-255> manner. I'll have a look into if the net modules have been separated enough that it can be formatted with the built in function.

Reply:
There is no problem in using an actual IP address but I think something like `127.0.0.1` could be nice as a place holder.